build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -64,13 +59,93 @@ limitations under the License.
   </tr>
 </table>
 
-## Behavior
+<!--
+  If the component is relevant for more than one format and operates differently between these
+  formats, include and filter multiple code samples.
+-->
+
+[filter formats=“websites”]
+
+[example preview=“inline” playground=“true” imports=“amp-component-name”]
+
+```html
+<amp-component-name required-attribute>
+  I am a hello world inline executable code sample for websites!
+</amp-component-name>
+```
+
+[/example][/filter]
+
+[filter formats=“ads”][example preview=“inline” playground=“true” imports=“amp-component-name”]
+
+```html
+<amp-component-name required-attribute>
+  I am a hello world inline executable code sample for ads!
+</amp-component-name>
+```
+
+[/example][/filter]
+
+## Usage
+
+One to three paragraph explaining the component usage.
+
+List important functionality.
+
+Explain why developers care about it.
```

Comment:
nit: merge all three lines into a single paragraph. Otherwise it gets tedious to remove them.

Reply:
ping